openqabot/types/incidents.py

```diff
@@ -134,8 +136,13 @@ def __call__(
                     if inc.rrid:
                         full_post["openqa"]["RRID"] = inc.rrid
 
+                    # Settings for the POST command
+                    full_post["openqa"].update(flavor_settings)
+                    full_post["openqa"]["VERSION"] = flavor_settings["VERSION"]
+                    full_post["qem"]["version"] = flavor_settings["VERSION"]
+                    full_post["openqa"]["DISTRI"] = flavor_settings["DISTRI"]
```

Comment:
```suggestion
```

Also redundant.

Reply:
> Also redundant.

Thanks, good catch but was there even before this PR. I'll address it in a next PR